README.adoc

```diff
@@ -404,9 +417,9 @@ the Open Liberty Operator.
 The `applicationImage` parameter defines what container image is deployed as part of the `OpenLibertyApplication` CRD. 
 This parameter follows the `<project-name>/<image-stream-name>[:tag]` format.
 The parameter can also point to an image hosted on an external registry, such as Docker Hub. 
-The [hotspot=system file=1]`system` application is configured to use the [hotspot=sysImage file=1]`image` created from the earlier build. 
+The [hotspot=system file=1]`system` microservice is configured to use the [hotspot=sysImage file=1]`image` created from the earlier build. 
 
-One of the benefits of using `ImageStream` is that the operator redeploys the application when it detects a new image is pushed into the `ImageStream`.
+One of the benefits of using `ImageStream` objects is that the operator redeploys the application when it detects a new image is pushed into the `ImageStream`.
```

Comment:
is it correct to say:

One of the benefits of using `ImageStream` objects is that the operator redeploys the application when it detects a new image is pushed into the `ImageStream` objects

Seems strange but should be consistent if there is one meaning for  `ImageStream`

Reply:
Perhaps saying just, "One of the benefits of using `ImageStream` objects is that the operator redeploys the application when it detects a new image is pushed" or is this too vague?